assets/javascripts/discourse/initializers/preview-edits.js.es6

```diff
@@ -160,6 +160,15 @@ export default {
           this.$('.topic-intro').prependTo(this.$('.main-link'))
           this.$('.topic-title').prependTo(this.$('.main-link'))
         }
+
+        if (socialMediaStyle) {
```

Comment:
@pmusaraj This should be in the TopicList component and should be confined to the necessary logic for your new setting. See the existing logic in the TopicList component here: https://github.com/pmusaraj/discourse-topic-previews/blob/a43842bd007711252f99605abc9981ae748c7000/assets/javascripts/discourse/initializers/preview-edits.js.es6#L42